src/creature.cpp

```diff
@@ -565,22 +566,22 @@ bool Creature::dropCorpse(Creature* lastHitCreature, Creature* mostDamageCreatur
 				break;
 		}
 
-		Tile* tile = getTile();
+		auto tile = getTile();
```

Comment:
Could this be `const auto`
```suggestion
		const auto tile = getTile();
```